src/inntektskildetype.js

```diff
@@ -1,31 +1,49 @@
 const inntektskildetype = [
-  {
-    kode: 'ARBEIDSINNTEKT_FRA_NORGE',
-    term: 'Arbeidsinntekt fra Norge',
-  },
-  {
-    kode: 'NÆRINGSINNTEKT_FRA_NORGE',
-    term: 'Næringsinntekt fra Norge',
-  },
-  {
-    kode: 'INNTEKT_FRA_UTLANDET',
-    term: 'Inntekt fra utlandet',
-  },
-  {
-    kode: 'FN_SKATTEFRITAK',
-    term: 'Ansatt i FN med skattefritak',
-  },
-  {
-    kode: 'MISJONÆR',
-    term: 'Misjonær som skal arbeide i utlandet i minst to år',
-  },
-  {
-    kode: 'PENSJON_UFØRETRYGD',
-    term: 'Pensjon/uføretrygd',
-  },
-  {
-    kode: 'PENSJON_UFØRETRYGD_KILDESKATT',
-    term: 'Pensjon/uføretrygd det betales kildeskatt av',
-  },
+    {
+        kode: 'ARBEIDSINNTEKT_FRA_NORGE',
+        term: 'Arbeidsinntekt fra Norge',
+    },
+    {
+        kode: 'NÆRINGSINNTEKT_FRA_NORGE',
+        term: 'Næringsinntekt fra Norge',
+    },
+    {
+        kode: 'INNTEKT_FRA_UTLANDET',
+        term: 'Inntekt fra utlandet',
+    },
+    {
+        kode: 'FN_SKATTEFRITAK',
+        term: 'Ansatt i FN med skattefritak',
+    },
+    {
+        kode: 'MISJONÆR',
+        term: 'Misjonær som skal arbeide i utlandet i minst to år',
+    },
+    {
+        kode: 'PENSJON_UFØRETRYGD',
+        term: 'Pensjon/uføretrygd',
+    },
+    {
+        kode: 'PENSJON_UFØRETRYGD_KILDESKATT',
+        term: 'Pensjon/uføretrygd det betales kildeskatt av',
+    },
+    {
```

Comment:
Legge inn en kommentar?
F.eks: 
// Inntektskilder for ftrl frivillig medlemskap
// Inntektskilder for pliktig medlemskap